src/substrait/textplan/tests/ParseResultMatchers.cpp

```diff
@@ -64,6 +65,44 @@ bool StringEqSquashingWhitespace(
   return atHave == have.end() && atExpected == expected.end();
 }
 
+std::vector<std::string> symbolNamesWithTypes(
+    const std::vector<std::shared_ptr<SymbolInfo>>& symbols,
+    const std::set<SymbolType>& types) {
+  std::vector<std::string> names;
+  for (const auto& symbol : symbols) {
+    if (types.find(symbol->type) != types.end()) {
+      names.push_back(symbol->name);
+    }
+  }
+  return names;
+}
+
+std::vector<std::string> OrderedSetDifference(
```

Comment:
Why ordered?  Isn't this just "set difference" and the fact that it uses an ordered set is an implementation detail?

Reply:
I chose ordered because the difference is displayed in the same order that the values were provided for comparison.  (As opposed to being in a random or sorted order.). This helps me see which ones are missing in the way the program should be processing them.